akka-stream/src/main/scala/akka/stream/io/IOResult.scala

```diff
@@ -0,0 +1,33 @@
+package akka.stream.io
+
+import akka.Done
+import scala.util.{ Failure, Success, Try }
+
+/**
+ * Holds a result of an IO operation.
+ *
+ * @param count A number of IO operations performed.
```

Comment:
then please document it that this value is context dependend. In general people are expected in bytes (especially since rechunking might happen). Which case needs something else than bytes?